Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,88 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+* Specify the namespace in which Cilium is installed as ``CILIUM_NAMESPACE``
+  environment variable. Subsequent commands reference this environment variable.
+
+  .. parsed-literal::
+
+      export CILIUM_NAMESPACE=kube-system
+
+  .. note::
+
+      Cilium is typically installed in ``kube-system`` namespace, but there are
+      a few exceptions such as GKE which install Cilium in a dedicated ``cilium``
+      namespace.
+
+* Hubble can be configured to be in **local mode** or **distributed mode (beta)**.
+
+  .. tabs::
+
+     .. group-tab:: Local Mode
+
+        In **local mode**, Hubble listens on a UNIX domain socket. You can connect to a
+        Hubble instance by running ``hubble`` command from inside the Cilium pod. This
+        provides networking visibility for traffic observed by the local Cilium agent.
+
+        .. parsed-literal::
+
+           helm upgrade cilium |CHART_RELEASE| \\
+              --namespace $CILIUM_NAMESPACE \\
+              --set global.hubble.enabled=true \\
+              --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}"
+
+     .. group-tab:: Distributed Mode (beta)
+
+
+        In **distributed mode (beta)**, Hubble listens on a TCP port on the host network.
+        This allows Hubble Relay to communicate with all the Hubble instances in the cluster.
```

Comment:
We probably want to briefly introduce Hubble Relay before mentioning it.

Reply:
added a link to https://docs.cilium.io/en/latest/hubble/#hubble-relay